### DISCUSSION TITLE
Game soft locks with no music

### DISCUSSION THREAD

**Discussion by praguzz** [ppy/osu#22031] [Ideas]
### Type

Other

### Bug description

The game suddenly plays no music at all in beatmap selection screen. Also, can't start or edit any beatmap whatsoever.

### Screenshots or videos

https://user-images.githubusercontent.com/15919651/210699760-3ff5593d-97d6-4e89-a951-bea7dfa341ca.mp4

### Version

2022.1228.0

### Logs

[database.log](https://github.com/ppy/osu/files/10349221/database.log)
[input.log](https://github.com/ppy/osu/files/10349222/input.log)
[network.log](https://github.com/ppy/osu/files/10349224/network.log)
[performance.log](https://github.com/ppy/osu/files/10349225/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10349226/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10349227/updater.log)

**Comment by peppy** [comment]
Are you able to capture a video from before it starts happening? Also if you open up the music player and play/pause does it have any effect?

This initially looks like it could be something at a low level, ie. audio driver not working as expected, but it's hard to say from only seeing a video from after it's already entered the bad state.

**Comment by praguzz** [comment]
Unfortunately, no, after I restart the game, it fixed itself and I haven't been able to reproduce it so far. I'll get back to you if it happens again.

Other sound effects still worked, so I wasn't thinking of audio driver issue.

**Reply by X9VoiD** [reply]
This happened to me as well however it only ever happened twice. It might not help much but my environment was on linux, playing audio via pipewire-alsa, so I don't think it's a problem of the audio driver. It might be within o!f.

**Comment by peppy** [comment]
Will convert this to an discussion for now, please let us know if you have it happen again.
